core/src/main/resources/springWAS.xml

```diff
@@ -24,6 +24,7 @@
 		factory-method="getTransactionManager"
 	/>
 
+<!-- Disable WebSphereMsSqlServerDbmsSupport temporarily, to see if BLOB/CLOB handling can be supported without temporary files
```

Comment:
waar is dit voor?

Reply:
Dit was omdat destijds de BLOBs en CLOBs het niet deden in WebSphere, omdat updateClob() het niet deed. Ik denk dat dat inmiddels is verholpen in WebSphere en/of de JDBC driver, of dat er sowieso een andere JDBC driver is, die het probleem niet heeft, waardoor de 'fix' niet meer nodig is.